client/src/css/Home.scss

```diff
@@ -1 +1,82 @@
 @import '../styles/colors.scss';
+
+.sessionCard {
+  box-shadow: none !important;
+  text-align: left !important;
+  width: 400px !important;
+  height: 100px !important;
+  margin-bottom: 20px !important;
+  border-radius: 15px !important;
+  transition: all 0.3s ease !important;
+  // box-shadow: 3px 3px 6px grey !important;
+}
+
+.ui.card > .content > .header {
+  font-family: Tommy !important;
+}
+.sessionCard:hover {
+  transition: all 0.3s ease;
+  background-color: #b2caee !important;
+  box-shadow: 3px 3px 6px grey !important;
+}
+
+.ui.button {
+  font-family: Tommy;
+  padding: 5px 27px;
+  margin-left: 280px;
+  transition: all 0.3s ease;
+}
+
+.homeHeader {
+  margin-top: 50px;
+  margin-left: 100px;
+  margin-bottom: 50px;
```

Comment:
If you're setting multiple margins, you can set them in one line like this:

`  margin: 50px 0px 50px 100px; `

The syntax is margin: margin-top value, margin-right value, margin-bottom value, margin-left value. (ie - starts from margin-top and goes clockwise)